src/main/java/com/github/xuanyu66/tidbcloudtoolkit/backend/service/TiDBCloudToolkitService.java

```diff
@@ -0,0 +1,114 @@
+package com.github.xuanyu66.tidbcloudtoolkit.backend.service;
+
+import com.github.xuanyu66.tidbcloudtoolkit.backend.entity.*;
+import com.github.xuanyu66.tidbcloudtoolkit.backend.exception.ToolkitException;
+import com.github.xuanyu66.tidbcloudtoolkit.backend.util.GsonUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class TiDBCloudToolkitService {
```

Comment:
implement interface and add code comment for both method end request objects and response objects